### PR TITLE
Add domain names to the Top 20 Now chart

### PR DESCRIPTION
**Pull request by JoshData**
The Top 20 Now chart only shows page titles. Some are mysterious without any context (e.g. "Forms & Pubs"; "Get Transcript"; "Application Manager"; "Home - eBenefits").

This commit adds the domain name of the page into the chart in smaller text to the right of the page title, but still within the link, to give an idea where the page link would take you. See the image below. The title text is modified in a corresponding way.

I'm adding a space after the page title to separate it from the <span> so that screen readers see a word break (just guessing here).

image

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -67,6 +67,12 @@
             suffix = n >= 12 ? "p" : "a";
         return (n % 12 || 12) + suffix;
       },
+      domainFromPage = function(page) {
+        // page is a URL without a scheme, return just the domain before the '/'
+        var i = page.indexOf('/');
```
Comment (konklone) [inline js/index.js]: Is it possible to have a page here with no trailing slash? (Domain root?)

File: js/index.js [hunk 2]
```diff
@@ -226,14 +232,19 @@
...
-              return d.page_title;
+              return d.page_title + " (" + domainFromPage(d.page) + ")";
             })
             .attr("href", function(d) {
               return exceptions[d.page] || ("http://" + d.page);
             })
             .text(function(d) {
-              return title_exceptions[d.page] || d.page_title;
-            });
+              return (title_exceptions[d.page] || d.page_title) + " ";
```
Comment (konklone) [inline js/index.js]: I think you could avoid putting in the `" "` by giving `span.domain` an attribute of `display: inline-block` and then using `padding` around it.

**Comment by konklone** [comment]
I added a couple technical notes, but since this is a UX change, I'd love some input from any of @cew821 @rypan @shawnbot @tdlowden @smarina04 on this one.

File: js/index.js (hunk 1 above)
Comment (JoshData) [inline js/index.js]: Is that a question for me? (I wouldn't know.)

**Comment by konklone** [comment]
Er, this got left sitting for a while. Anyone have an opinion on whether this should go up on the site?

**Pull request closed without merging**